sysl2/sysl/ValueUtil.go

```diff
@@ -2,6 +2,7 @@ package main
 
 import (
 	sysl "github.com/anz-bank/sysl/src/proto"
+	"github.com/anz-bank/sysl/sysl2/sysl/syslutil"
```

Comment:
Its great having a util package where we can aim for 100% coverage.
A facade to the whole sysl library would be nice too. Would I have to import multiple packages?

Reply:
Agreed. This is just a first step so I can consume the library from somewhere other than sysl main.